packages/world/src/modules/std-delegations/DisposableDelegationControl.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { DelegationControl } from "../../DelegationControl.sol";
+import { DisposableDelegations } from "./tables/DisposableDelegations.sol";
+
+contract DisposableDelegationControl is DelegationControl {
```

Comment:
I like this better than "decaying" 👍 

Another potential name idea is "callbound" (to go with "timebound")